src/chroma/distance.nim

```diff
@@ -0,0 +1,66 @@
+## Implementation of CIEDE2000 color difference formula
+## 
+## See: https://en.wikipedia.org/wiki/Color_difference#CIEDE2000
+## 
+## The implementation follows: http://www2.ece.rochester.edu/~gsharma/ciede2000/ciede2000noteCRNA.pdf
+
+import colortypes, transformations, math
+
+const twentyfiveToSeventh = (25^7).float32
+
+proc myAtan(x, y: float32): float32 =
+  if x == 0 and y == 0:
+    return 0
+  elif x >= 0:
+    arctan2(x, y).radToDeg
+  else:
+    arctan2(x, y).radToDeg + 360
+
+func deltaE00*(c1, c2: ColorLAB, k_L, k_C, k_H = 1.float32): float32 =
```

Comment:
`k_L` --> `kL`

Please follow Nim naming convention - no "_".

Reply:
done. I still have in that file some variable names that start with a Capital letter (although convention would want that only for types). Do you want me to adjust those too?

Reply:
Yes please.